src/WrapperApp/components/Results/ResultsPanel.tsx

```diff
@@ -26,20 +16,35 @@ export interface EstimatorResults extends Estimator {
 
 function ResultsPanel() {
 	const { open: openSaveFileDialog } = useDialog('saveFile');
-	const { yaptideEditor, resultsSimulationData: simulation } = useStore();
+	const {
+		yaptideEditor,
+		resultsSimulationData: simulation,
+		setResultsSimulationData
+	} = useStore();
 
 	const [tabsValue, setTabsValue] = useState(0);
 	const [estimatorsResults, setEstimatorsResults] = useState<EstimatorResults[]>([]);
 	const [groupQuantities, setGroupQuantities] = useState(false);
 
+	const inputFilesEstimatorNames = simulation?.input.inputFilesEstimatorNames?.map(output =>
```

Comment:
can you rename inputFiles part to userInputFiles ?

Reply:
also - I do not like the idea that `simulation?.input.inputFilesEstimatorNames` is being created and filled with conent in deeply nested part of `findFirstEstimatorName` callback:
https://github.com/yaptide/ui/blob/5c71aafe83cfe16102c19afd3750986bb53b0826/src/services/ShSimulatorService.tsx#L559

Reply:
this method should be well documented with **comments** so the **reader of the code** knows what happens when we load results of the simulation which has already completed execution and what happens if we load some type of JSON like project with results (like example)